From 7bf2fc388df10bc1760cd06594f50da0c78ee3a2 Mon Sep 17 00:00:00 2001
From: fidomax <adobegitler@gmail.com>
Date: Tue, 26 Jan 2021 07:33:17 +0300
Subject: [PATCH] Add HAVE_VASPRINTF HAVE_ASPRINTF templates

It fixes autoreconf error:
autoheader: warning: missing template: HAVE_ASPRINTF
autoheader: Use AC_DEFINE([HAVE_ASPRINTF], [], [Description])
autoheader: warning: missing template: HAVE_VASPRINTF

Signed-off-by: Maxim Kochetkov <fido_max@inbox.ru>
Fetch from: https://github.com/postgis/postgis/commit/d0ac5df49961d1d1a74b894f9941b973c0695c11.patch
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 9ac7051bb..dddb3aef7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11,6 +11,8 @@ dnl **********************************************************************
 
 AC_INIT()
 AC_CONFIG_HEADERS([postgis_config.h])
+AH_TEMPLATE([HAVE_VASPRINTF])
+AH_TEMPLATE([HAVE_ASPRINTF])
 AC_CONFIG_MACRO_DIR([macros])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_PROG_INSTALL
-- 
2.30.0

